src/components/Fields/FieldRate.tsx

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+import {
+  type Control,
+  type FieldPath,
+  type FieldValues,
+} from "react-hook-form";
+
+import { Rate } from "@/components";
+import { Label } from "@/ui";
+import { FormControl, FormField, FormItem } from "@/ui/form";
```

Comment:
```suggestion
import { FormControl, FormField, FormItem } from "@/ui";
```
Чтобы вот так выглядел импорт